core/src/test/java/com/graphhopper/routing/util/spatialrules/SpatialRuleLookupArray_DiffblueTest.java

```diff
@@ -0,0 +1,130 @@
+/**
+ *
+ *
+ * Copyright 2018 Diffblue Limited
+ *
+ * Diffblue Limited licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.graphhopper.routing.util.spatialrules;
+
+import com.diffblue.deeptestutils.Reflector;
+import com.graphhopper.routing.util.spatialrules.SpatialRule;
+import com.graphhopper.routing.util.spatialrules.SpatialRuleContainer;
+import com.graphhopper.routing.util.spatialrules.SpatialRuleLookupArray;
+import com.graphhopper.util.shapes.BBox;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class SpatialRuleLookupArray_DiffblueTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  /* testedClasses: SpatialRuleLookupArray */
+
+  /*
+   * Test generated by Diffblue Deeptest.This test covers `boolean
+   * isBorderTile(int, int, int)' block 14 (line 141)
+   *
+   */
+
```

Comment:
These two I'm not sure, lots of reflections but it is for a private method. @peterschrammel @petersear and @James-Adam what do you think.

Reply:
👎  reflective construction is no good here.

Reply:
Can you create a bug ticket for this as a challenge to instantiate this non-reflectively?